src/App.js

```diff
@@ -40,12 +58,27 @@ class Container extends React.Component {
   }
 
   render() {
+    var layout = [
```

Comment:
I think this should be `const`. It should probably be defined somewhere else too.

Reply:
Updated. Will leave it there until we find a better place for it.